CondFormats/BeamSpotObjects/interface/BeamSpotOnlineObjects.h

```diff
@@ -55,9 +59,42 @@ class BeamSpotOnlineObjects : public BeamSpotObjects {
   // set number of Primary Vertices used in the BeamSpot fit
   void SetNumPVs(int val);
 
+  // set number of Events used in the BeamSpot fit (for DIP)
+  void SetUsedEvents(int val);
```

Comment:
according to rule 2.8 https://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1 the method names should be lowercase (I know it was done for consistency with pre-existing ones, but nonetheless)

Reply:
It was done more to be in line with the already existing [BeamSpotObjects.h](https://github.com/cms-sw/cmssw/blob/master/CondFormats/BeamSpotObjects/interface/BeamSpotObjects.h), but ok I will change the code according to the rules.

Reply:
Actually going back to this I don't think I will change it now, because `BeamSpotOnlineObjects` inherit most of its methods from `BeamSpotObjects` which has methods with uppercase. I prefer to change them all at once in a later PR since also all the consuming code for both `BeamSpotOnlineObjects` and `BeamSpotObjects`)has to be tracked down and migrated.